File: Data Structures & Algorithms/is-palindrome/submission-0.py
class Solution:
    def isPalindrome(self, s: str) -> bool:
        cleaned = re.sub(r'[^a-z0-9]', '', s.lower())
        start = 0
        end = len(cleaned) - 1

        while (end > start):
            if (cleaned[start] != cleaned[end]):
                return False
                
            start += 1
            end -= 1
        
        return True
        

File: Data Structures & Algorithms/arranging-coins/submission-0.ts
class Solution {
    /**
     * @param {number} n
     * @return {number}
     */
    arrangeCoins(n: number): number {
        let start = 1;

        while (start <= n) {
            n -= start;
            start++;
        }

        if (start === n) return start;
        else return start - 1;
    }
}
